bootstrap/spigot/src/main/java/org/geysermc/geyser/platform/spigot/GeyserSpigotPlugin.java

```diff
@@ -172,7 +169,30 @@ public void onEnable() {
             return;
         }
 
-        this.geyserCommandManager = new GeyserSpigotCommandManager(geyser);
+        PaperCommandManager<GeyserCommandSource> cloud;
+        try {
+            cloud = new PaperCommandManager<>(
+                this,
+                CommandExecutionCoordinator.simpleCoordinator(),
+                SpigotCommandSource::new,
+                source -> (CommandSender) source.handle()
+            );
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+
+        try {
+            // Should always be available on 1.13 and up
+            cloud.registerBrigadier();
+        } catch (BukkitCommandManager.BrigadierFailureException e) {
+            geyserLogger.error("Failed to initialize Brigadier support: " + e.getMessage());
```

Comment:
I still don't think most users need to care about this for it to be an error. It'll just clog up support channels and most people won't notice.

Reply:
ah yeah i just forgot 👍